bip-powx-low-energy-pow.mediawiki

```diff
@@ -239,6 +271,11 @@ Although it is out of the scope of this proposal, the authors strongly recommend
 
 A hard fork is not necessarily required for the Bitcoin network to test and eventually implement oPoW. It’s possible to add oPoW as a dual PoW to Bitcoin as a soft fork. Tuning the parameters to ensure that, for example, 99.9% of the security budget would be earned by miners via the SHA256 Hashcash PoW and 0.1% via oPoW would create sufficient incentive for oPoW to be stress-tested and to incentivize the manufacture of dedicated oPoW miners. If this test is successful, the parameters can be tuned continuously over time, e.g. oPoW share doubling at every halving, such that oPoW accounts for some target percentage (up to 100% in a complete SHA256 phase-out).
 
+
+==== Reverse compatibility ====
+
+In general, we do not claim reverse compatibility, meaning it may not be possible to fully implement oPoW and maintain compatibility with older versions of the codebase that only perform SHA256 mining.
```

Comment:
"do not claim" "may not" is not clear. This section shouldn't be about claims, but facts - how is it compatible? how is it not?

It's probably fine to be uncertain on miner implementations, but absolutes should be known and stated explicitly insofar as the consensus protocol is concerned.

Reply:
oPoW will not be reverse compatible

Reply:
Ok, please say so explicitly in the BIP :)

Reply:
Sure https://github.com/bitcoin/bips/pull/1126/commits/ff47e7ef144224dbd94c6c045f7ca1bffa04e98a ;)